### PR TITLE
Capstone: motorcycle appointment app

### PR DESCRIPTION
**Pull request by Adams-Ghub**
## The following were carried out in this project

- Create a register page to allow users to create an account with a unique username.
- Create a login page to allow users to login
- Create a navigation panel where the user can see all links to the following pages
    - Motorcycles
    - Reserve
    - My reservations
    - Add Motorcycles
    - Delete
 - Display all motorcycles on the Motorcycles/Home page
 - Display details of a selected motorcycle on the Details page with a button to the reserve form
 - Display form for adding a motorcycle with the related details.
 - Display form to reserve a motorcycle with the following features
    - User and Model are auto-filled if the button on the details page is clicked. 
    - User is auto-filled if the button on the navigation bar is clicked.
 - Display a list of all reservations that have been made in the Reservations page
 - Display a list of all motorcycles with title and Delete button to delete a motorcyle.
 - Made the project responsive using mobile-first approach
 
[link to back-end pull request](https://github.com/Bennyjoez/motorcycle-appointment-app-backend/pull/19)

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+import Register from './components/sessions/register';
+import Login from './components/sessions/login';
+import Motorcycles from './components/pages/motorcycles';
+import Reservations from './components/pages/reservations';
+import AddMotorcycle from './components/pages/add-motorcycle';
+import Delete from './components/pages/delete';
+import Reserve from './components/pages/reserve';
+import Details from './components/pages/details';
+
+function App() {
```
Comment (Meltrust) [inline src/App.js]: - [x]  Please, kindly refactor this and all other functions in your project to ES6 arrow functions, so your code complies with the latest standards.

Refactoring is easy:

```javascript
const App = () => {
  // ...
}
```

## Explanation

The `function` keyword has been soft deprecated in almost all the software industry in favor of ES6 arrow functions. This means, almost nobody uses it anymore, so much that it's considered a non-best practice.

You can do a search in Vscode for the "function" word, and correct as needed.  Shouldn't take you more than 5-10 min so don't worry (I found only 4):

image


That way, your code will meet the latest standards. 💪

**Review by Meltrust — CHANGES_REQUESTED** [review]
Hi @Adams-Ghub,

Good job so far!
There are some issues that you still need to work on to prepare your project for the final evaluation, but you are almost there!






### To highlight:

-  Tests are passing✔️
-  Nice code organization ✔️ 
-  App is working well✔️ 
-  Good readme ✔️ 
-  Beautiful frontend ✔️ 



#### You are really close to finishing the Microverse program!! Keep it up! 👍👍👍

You can do it

### After implementing the requested changes, please submit another review request. ♻️
_Check the comments under the review._







Cheers and Happy coding!👏👏👏

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the previous reviews unless it is requested otherwise._**

------

**Review by Meltrust — APPROVED** [review]
#### Hi @Adams-Ghub,


# **Wow, you did it 🎉**
Brilliant
## Thank you for the changes implemented 💪 🥇 ㊗️ 
#### _Unless you want to add more features, go ahead to your final presentation ⏩ ⏩ ⏩_ 
## You are about to finish the Microverse program.  You have come a long way!!!




### Good luck in the software industry!!  I'll see you there. ✨
### **Congratulations!!!!!!**  🎉
applause




Cheers and Happy coding!👏👏👏


------

**Comment by Bennyjoez** [comment]
Thank you very much @Meltrust We sure are happy 😄 🕺🏾 👯 

@Adams-Ghub @Mikey1-nuel @harshi0102 Bravoo 👏🏾 👏🏾

**Pull request merged**